bundles/org.openhab.binding.mqtt.generic/src/main/resources/ESH-INF/config/trigger-channel-config.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:mqtt:trigger_channel">
+		<parameter-group name="transformations">
+			<label>Transform Values</label>
+			<description>These configuration parameters allow you to alter a value before it is published to MQTT or before a
```

Comment:
"before it is published to MQTT"? A trigger channel cannot be used to send any information outwards - I guess this statement should be removed.